composer.json

```
@@ -147,5 +147,11 @@
         }
     },
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/tomasVotruba/php-parser"
```

you need to define `"no-api": true` here:

```suggestion
            "url": "https://github.com/tomasVotruba/php-parser",
            "no-api": true
```

I wrote a blog post about it https://samsonasik.wordpress.com/2018/10/07/using-no-api-key-on-github-forked-repository-in-composer-repositories-configuration-for-travis-build/